src/components/footer/footer--big.njk

```diff
@@ -68,7 +68,7 @@
             {% for item in secondary_section.contact_links.social_nav %}
               <div class="grid-col-auto">
                 <a class="usa-social-link usa-social-link--{{ item | lower }}" href="javascript:void(0);">
```

Comment:
Same:

```suggestion
                <a class="usa-social-link usa-social-link--{{ item.name | lower }}" href="javascript:void(0);">
```

Reply:
Ah yes, this would fix the issue — but the bigger problem is that this code should match the code from `footer.njk` and use an regular image, not a background image. cc @mejiaj 

```suggestion
                <a class="usa-social-link" href="javascript:void(0);">
                  <img class="usa-social-link__icon"
                    src="{{ uswds.path }}/img/usa-icons/{% if item.icon %}{{ item.icon }}{% else %}{{ item.name | lower }}{% endif %}.svg"
                    alt="{{ item.name }}"
                  />
```